.travis.yml

```diff
@@ -58,7 +58,7 @@ env:
 before_script:
   - mkdir build
   - cd build
-  - cmake -DBUILD_TESTS=ON -DBUILD_PYTHON=ON -DERT_BUILD_CXX=ON ..
+  - cmake -DBUILD_TESTS=ON -DBUILD_PYTHON=ON -DERT_BUILD_CXX=ON -DBUILD_APPLICATIONS=ON ..
```

Comment:
Applications is just a set of small utilities for working with binary eclipse files, could probably to a large extent be replaced by 10 line Python scripts.